test/ably/restchannelpublish_test.py

```diff
@@ -59,7 +57,8 @@ def test_publish_various_datatypes_text(self):
         message_contents = dict((m.name, m.data) for m in messages)
         log.debug("message_contents: %s" % str(message_contents))
 
-        self.assertEqual(True, message_contents["publish0"],
+
+        self.assertEqual("true", message_contents["publish0"],
                 msg="Expect publish0 to be Boolean(true)")
         self.assertEqual(24, int(message_contents["publish1"]),
```

Comment:
We don't support literal content types other than String, Binary and JSON objects